components/ProjectForm/index.js

```diff
@@ -34,103 +34,98 @@ export default function ProjectForm({ projects, onAddProject }) {
 
   return (
     <>
-      <Header />
-      <StyledSection>
-        <StyledForm onSubmit={handleSubmit(onSubmit)}>
-          <h2>Create new project</h2>
+      <StyledForm onSubmit={handleSubmit(onSubmit)}>
+        <h2>Create new project</h2>
+        <label>
+          Project title
+          <input
+            {...register("title", { required: "Title is required" })}
+            placeholder="Title"
+          />
+          <StyledErrorMessage>{errors.title?.message}</StyledErrorMessage>
+        </label>
+
+        <label>
+          Image
+          <input {...register("image")} placeholder="Select Image" />
+        </label>
+
+        <label>
+          Description
+          <textarea
+            {...register("description", {
+              required: "Description is required",
+            })}
+            placeholder="Description"
+          />
+          <StyledErrorMessage>{errors.description?.message}</StyledErrorMessage>
+        </label>
+        <div>
           <label>
-            Project title
-            <input
-              {...register("title", { required: "Title is required" })}
-              placeholder="Title"
-            />
-            <StyledErrorMessage>{errors.title?.message}</StyledErrorMessage>
+            Duration
+            <select {...register("duration")} defaultValue={"select..."}>
```

Comment:
it would be more common to use an empty string as a default value. The display text for the option is of course fine.